src/cata_string_consts.h

```diff
@@ -49,6 +49,8 @@ static const activity_id ACT_HACKING( "ACT_HACKING" );
 static const activity_id ACT_HACKSAW( "ACT_HACKSAW" );
 static const activity_id ACT_HAIRCUT( "ACT_HAIRCUT" );
 static const activity_id ACT_HAND_CRANK( "ACT_HAND_CRANK" );
+static const activity_id ACT_HAND_PUMP_INTEGRAL( "ACT_HAND_PUMP_INTEGRAL" );
```

Comment:
don't use this file. this was a mistake that got reverted mostly. it increases compile times and increases ccache hits

Reply:
It's OK for me to just delete that line? It won't break anything?